Frames.podspec

```diff
@@ -15,9 +15,7 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Source/**/*.swift'
   s.exclude_files = "Classes/Exclude"
-  s.resource_bundles = {
-    'Frames' => ['Source/Resources/**/*']
-  }
+  s.resources = 'Source/Resources/**'
```

Comment:
Just curious, What does this fix do ? I see it includes recursively everything in Resources folder, How is that from previous change ?